lib/vanagon/driver.rb

```diff
@@ -33,6 +35,19 @@ def initialize(platform, project, options = { :configdir => nil, :target => nil,
       raise Vanagon::Error.wrap(e, "Could not load the desired engine '#{engine}'")
     end
 
+    def filter_out_components(project, only_build)
+      filtered_components = []
+      only_build.each do |comp|
+        filtered_components += @project.filter_component(comp)
```

Comment:
Should this be `@project` (instance variable) or `project` (passed in argument to this method)? Which one are you intending to call? Consider a more descriptive variable name to avoid conceptual (if not literal) shadowing?